src/ProductTour/Checkpoint.jsx

```diff
@@ -117,8 +117,14 @@ const Checkpoint = React.forwardRef(({
         {...props}
       />
       <div id="pgn__checkpoint-arrow" data-popper-arrow />
-      {/* This text is not translated due to Paragon's lack of i18n support */}
-      <span className="sr-only">Bottom of step {index + 1}</span>
+      <span className="sr-only">
+        <FormattedMessage
+          id="pgn.ProductTour.Checkpoint.bottom-position-text"
+          defaultMessage="Bottom of step {step}"
+          values={{ step: index + 1 }}
+          description="Screen-reader message to notify user that they are located at the bottom of the product tour step."
+        />
+      </span>
```

Comment:
I propose to put everything related to translations (keys, descriptions, identifiers) into a separate `messages.js` file. We already have an example of how we did this in the [ChipCarousel](https://github.com/openedx/paragon/blob/master/src/ChipCarousel/messages.js) component. This will simplify the component code and make it thinner.

It may be worth considering this improvement in a separate task 🥇

Reply:
Yeah, sounds good, updated, thanks